test/components/selection-list.spec.tsx

```diff
@@ -1,31 +1,28 @@
 import keycode = require('keycode');
 import * as React from 'react';
-import {ClientRenderer, expect, simulate, sinon} from 'test-drive-react';
+import {ClientRenderer, DriverBase, expect, sinon} from 'test-drive-react';
 import {SelectionListDemo} from '../../demo/components/selection-list-demo';
-import {divider, SelectionList} from '../../src/components/selection-list';
-import baseStyle from '../../src/components/selection-list/selection-list.st.css';
+import {divider, Option, SelectionList} from '../../src/components/selection-list';
```

Comment:
I think you should import from `../../src`. naming of exports should be more specific